wp-content/themes/hurumap/acf-json/group_5eafdb1130355.json

```diff
@@ -42,8 +42,8 @@
                 },
                 {
                     "key": "field_5eafdb84e2bc4",
-                    "label": "Image",
-                    "name": "image",
+                    "label": "Logo",
+                    "name": "logo",
```

Comment:
this affects Outbreak, I am going back to make sure Outbreak remains the same, and Promise Tracker changes instead because this is on Promise Tracker.